.circleci/config.yml

```diff
@@ -756,6 +756,10 @@ workflows:
           name: build-discovery-provider
           repo: discovery-provider
 
+      - docker-build-and-push:
+          name: build-network-monitoring
+          repo: discovery-provider/plugins/network-monitoring
```

Comment:
Dockerhub doesn't allow `/` in the name, that's probably why this is failing.

Try `network-monitoring` should work